src/net_server.c

```diff
@@ -383,11 +383,35 @@ static net_client_t *NET_SV_Controller(void)
     return best;
 }
 
+typedef enum
+{
+    RANGE_LOCALHOST,   // Same process or 127.x
+    RANGE_PRIVATE,     // RFC 1918
+    RANGE_PUBLIC,      // The public Internet
+} ip_range_t;
+
+static ip_range_t ClientAddressRange(const char *addr)
+{
+    if (!strcmp(addr, "local client")
+     || M_StringStartsWith(addr, "127."))
+    {
+        return RANGE_LOCALHOST;
+    }
+    if (!M_StringStartsWith(addr, "10.")
```

Comment:
Is this correct or a copy-paste error from above?

Reply:
Whoops, it absolutely is a mistake. Now fixed.